packages/agoric-cli/src/deploy.js

```diff
@@ -326,7 +326,10 @@ export default async function deployMain(progname, rawArgs, powers, opts) {
             const fileName = paths.pop();
             try {
               return require.resolve(fileName, {
-                paths: [...paths, path.dirname(moduleFile)],
+                paths: [
+                  path.resolve(path.dirname(moduleFile), ...paths),
```

Comment:
This is more of a drive-by, as I believe the spread was a mistake: for `path.resolve`, `paths` components would be joined if not absolute, where for `require.resolve`, each `paths` is a new root. Since I believe `dirname(moduleFile)` is absolute, this should result in the correct behavior and never use `cwd`.

Reply:
This was introduced in https://github.com/Agoric/agoric-sdk/pull/5132/commits/d83dc644d91c50b2ab5bbcb45c547956db9fec3f and I didn't see a reasoning behind the spread.

Reply:
Thanks for catching this!